test/upgrade/create-in-v0.7.3-special-functions.td

```diff
@@ -20,8 +20,12 @@
 # Do not allow NOW / CURRENT_TIMESTAMP to be persisted at all
 # if they ever are allowed, add an upgrade test for them
 
+# The error message changed substantially in v0.20.0. The only commonality is
+# that the error message contains the disallowed function, so that is what we
+# check for.
```

Comment:
this is an interesting usecase for something like `skip-if`, perhaps we support multiple `contains` or something